api/v1alpha1/etcdcluster_webhook.go

```diff
@@ -106,3 +121,66 @@ func (r *EtcdCluster) ValidateDelete() (admission.Warnings, error) {
 	etcdclusterlog.Info("validate delete", "name", r.Name)
 	return nil, nil
 }
+
+// validatePdb validates PDB fields
+func (r *EtcdCluster) validatePdb() (admission.Warnings, field.ErrorList) {
```

Comment:
Internally discussed and agreed that pdb percentage validation and creation is missing. Waiting for the implementation.